rust/src/http2/http2.rs

```diff
@@ -124,6 +128,15 @@ pub struct HTTP2Frame {
     pub data: HTTP2FrameTypeData,
 }
 
+#[derive(Debug, Default)]
+pub struct DohHttp2Tx {
```

Comment:
can you add field description comments, esp on the index to these 2 fields

Reply:
ok